migration-util/src/main/java/edu/unc/lib/dcr/migration/content/ContentObjectTransformer.java

```diff
@@ -363,7 +364,7 @@ private void populateFileObject(Resource bxc3Resc, Model depositModel) {
         // transform existing PREMIS as the events for the file object, rather than work
         transformPremis(originalPid, filePid);
         if (isNewWork) {
-            addMigrationEvent(getPremisLogger(newPid));
+            addMigrationEvent(getPremisLogger(newPid), depositBxc3.getFullname());
```

Comment:
if its a new work, then it was generated in bxc5, so you should need to override here